test/vm.install

```diff
@@ -4,9 +4,6 @@ set -eu
 
 # for Debian based images, build and install debs; for RPM based ones, the locally built rpm gets installed separately
 if [ -d /var/tmp/debian ]; then
-    apt-get update
-    eatmydata apt-get install ${APT_INSTALL_OPTIONS:-} -y cockpit-ws cockpit-system podman
```

Comment:
This was not only installing basic cockpit packages as the commit message suggests, but also podman package.
Do you know why that was there in the first place? Do we pre-install podman in our images actually or we rely on this here for cockpit-podman dependencies to exist?

Reply:
Our Debian/Ubuntu images have pre-installed podman since February: https://github.com/cockpit-project/bots/commit/1d94c0e7a88d -- for the same reason. I am happy to re-push to adjust the commit message, or we just do it on squash&merge?

Reply:
No need - thanks for the clarification.